### PR TITLE
Admin: Category product listing tweaks and validations

### PR DESCRIPTION
**Pull request by torvista**
show categories path on tab title (for identification of multiple tabs)
add popup link to catalog to page title: to open shop view for that category
fix empty class
alt text for linked category icon
new title for linked category icon
incorrect uses of label on radio group replaced with fieldset
renamed search form as clashed with input id "search" when used for label for
new css shrink style for td ID to reduce wasted space
alt tags for images
removed unused table columns for categories
removed button role from cat tr (invalid to have <a href inside button): added pointer for visual cue

### CODE REVIEW

**Comment by torvista** [comment]
personally I also show weights, and have a switch on the page to show images.
I could add those if warranted...?

File: admin/category_product_listing.php [hunk 1]
```diff
@@ -361,7 +361,7 @@
       function init() {
           cssjsmenu('navbar');
           if (document.getElementById) {
-              var kill = document.getElementById('hoverJS');
+              let kill = document.getElementById('hoverJS');
```
[the remark this answers is not in the document]
Comment (drbyte) [reply admin/category_product_listing.php]: It was suggested. Not decided.

**Comment by drbyte** [comment]
There's a LOT of stuff going on here. Very hard to do code review.

In future, please do separate PRs for separate kinds of changes. Fewer changes means easier to follow, therefore faster review, faster merge.

File: admin/category_product_listing.php [hunk 2]
```diff
@@ -352,7 +352,7 @@
 <html <?php echo HTML_PARAMS; ?>>
   <head>
     <meta charset="<?php echo CHARSET; ?>">
-    <title><?php echo TITLE; ?></title>
+    <title><?php echo 'Admin: ' . zen_output_generated_category_path($current_category_id); ?></title>
```
[the remark this answers is not in the document]
Comment (drbyte) [reply admin/category_product_listing.php]: Since `TEXT_ADMIN_TAB_PREFIX` is already defined as "Admin", let's use that here.

**Comment by torvista** [comment]
Yes I know. there is a lot of stuff. It's hard to compartmentalise this one...or rather do some stuff, then PR then wait, then a bit more...etc.

File: admin/category_product_listing.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (torvista) [reply admin/category_product_listing.php]: done

File: admin/category_product_listing.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (torvista) [reply admin/category_product_listing.php]: reverted.

**Comment by Zen4All** [comment]
@torvista You don't have to wait, you can make several PR at the same time.

**Comment by torvista** [comment]
I thought that would be worse for people checking it...having to revisit the file several times.

**Comment by Zen4All** [comment]
I understand the confusion, but it is often easier to review small/related changes in the code, then a bigger single PR with multiple unrelated changes.
Sometimes this means you will have to wait with your next PR, because these changes influence each other, and other times you can push multiple PR's at the same time.

**Pull request merged**